src/cmd/src/standalone.rs

```diff
@@ -71,6 +71,7 @@ pub struct StandaloneOptions {
     pub mode: Mode,
     pub wal_dir: String,
     pub storage: ObjectStoreConfig,
+    pub memory_catalog_enable: bool,
```

Comment:
enable_memory_catalog? Or use_memory_catalog, which means we force it to use memory catalog.